pages/api/health-check.ts

```diff
@@ -0,0 +1,5 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default (req: NextApiRequest, res: NextApiResponse): any => {
+    res.status(200).json({ status: "working" });
```

Comment:
Let's make this 'OK', instead of 'working'.

Reply:
done